src/main/java/com/sonymobile/tools/gerrit/gerritevents/workers/Coordinator.java

```diff
@@ -28,14 +28,15 @@
 import java.util.concurrent.BlockingQueue;
 
 /**
- * Representation interface of an EventThread workers coordinator.
+ * Representation interface of a Work coordinator.
  * @author Robert Sandell &lt;robert.sandell@sonyericsson.com&gt;
  */
 public interface Coordinator {
     /**
      * Retrieves the work queue for workers to poll.
      * @return the queue
      */
+    @Deprecated
```

Comment:
Needs a walkthrough of gerrit-trigger-plugin to figure out what it needed this for and probably add whatever it actually needs. A bit iffy in terms of binary compatibility. It might be time to go to java 8 and add default methods, or introduce a `Coordinator2` interface?

Reply:
Gerrit trigger plugin is not using the `getWorkQueue()` method.